lstchain/calib/camera/tests/test_calib.py

```diff
@@ -1,7 +1,9 @@
 import numpy as np
-from lstchain.calib import camera
+from lstchain.calib.camera.calib import gain_selection
+from astropy.utils import deprecated
 
 
+@deprecated('28/06/2019', message='gain selection is now performed at <= R1 calibration level')
```

Comment:
I would at least eliminate the tests

Reply:
The function is still used in the lst custom calibration code (that I don't know if some people use it and would anyway leave it until a proper release)